resources/META-INF/plugin.xml

```diff
@@ -252,6 +252,10 @@
               text="Flutter Packages Upgrade"
               description="Run 'flutter packages upgrade'"/>
       <separator/>
+      <action id="flutter.xcode.open" class="io.flutter.actions.OpenInXcodeAction"
+              text="Open in Xcode..."
```

Comment:
Assuming the menu in IntelliJ generally use the ellipses char, we should use that here (available a few lines down).